csrc/preseg_passes/remove_bcast_squeeze.cpp

```diff
@@ -0,0 +1,95 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <debug.h>
+#include <ir/utils.h>
+#include <options.h>
+#include <preseg_passes/remove_bcast_squeeze.h>
+#include <root_domain_map.h>
+
+namespace nvfuser::preseg_passes {
+
+namespace {
+inline bool isFusionOutput(Fusion* fusion, Val* val) {
```

Comment:
Have you considered https://github.com/NVIDIA/Fuser/blob/1e48b3752f8cfc976c83df5f31b6ddc2b12f6fc6/csrc/ir/base_nodes.h#L368?

Reply:
changed.